coffee_cmd/src/cmd.rs

```diff
@@ -77,6 +80,10 @@ impl From<&CoffeeCommand> for coffee_core::CoffeeOperation {
             CoffeeCommand::Remove { plugin } => Self::Remove(plugin.to_owned()),
             CoffeeCommand::Show { plugin } => Self::Show(plugin.to_owned()),
             CoffeeCommand::Nurse {} => Self::Nurse,
+            CoffeeCommand::Tip {
+                plugin,
+                amount_msat,
+            } => Self::Tip(plugin.to_owned(), amount_msat.clone()),
```

Comment:
```suggestion
            } => Self::Tip(plugin.to_owned(), *amount_msat),
```

Reply:
this is the same, I prefer `.clone()` in this way also the non C people understand that they are cloning the object